gbasis/base_one.py

```diff
@@ -17,34 +17,33 @@ class BaseOneIndex(BaseGaussianRelatedArray):
     ----------
     _axes_contractions : tuple of tuple of GeneralizedContractionShell
         Contractions that are associated with each index of the array.
-        Each tuple of GeneralizedContractionShell corresponds to an index of the array.
-
-    Properties
-    ----------
+        Each tuple of `GeneralizedContractionShell` corresponds to an index of the array.
     contractions : tuple of GeneralizedContractionShell
         Contractions that are associated with the first index of the array.
+        Property of `BaseOneIndex`.
 
     Methods
     -------
     __init__(self, contractions)
         Initialize.
     construct_array_contraction(self, contraction) : np.ndarray(M, L_cart, ...)
         Return the array associated with a `GeneralizedContractionShell` instance.
-        `M` is the number of segmented contractions with the same exponents (and angular momentum).
-        `L_cart` is the number of Cartesian contractions for the given angular momentum.
+        :math:`M` is the number of segmented contractions with the same exponents (and angular
+        momentum).
+        :math:`L_cart` is the number of Cartesian contractions for the given angular momentum.
```

Comment:
So the whole point of `:math:` is to let sphinx  (or whatever renderer) know to render the content as a latex equation rather than to let the user know that it is a math variable. In this case, `L_cart` as a latex equation will look like the following 
https://latex.codecogs.com/gif.latex?L_cart
which will need to be fixed when we start using sphinx.

Now I realized this change came about because the form of the docstring was being standardized, and some of the variables are treated as math equations and some treated as "code". I like the use of math equations as much as possible since it makes as consistent (the math equations are rendered differently from backquoted objects) with the equations that are present elsewhere, but this case doesn't really fit, unless you change the whole thing to ":math:`L_{\mathrm{cart}}`" which is a bit of mouthful. Also these terms will be rendered differently from `np.ndarray(M, L_cart, ...)`.

I think it'd be better if the `:math:` part is dropped here.

Also, I don't remember sure how much experience you have with Latex (I remember talking to you about it but I forgot), but if you aren't super comfortable with it, we can treat it as a normal code object, as fix it later when we start generating the html's.

Reply:
I had meant to only apply `:math:` to the math usages of variables, not the "code" usages, but I guess I was on autopilot and ended up losing sight of which ones I meant to change. 

I'm comfortable with Latex, but this is my first time using Sphinx, so for some reason that's been tripping me up, even though it's just being treated as a Latex equation.